### DISCUSSION TITLE
more BIP39-enhanced applications   8-)

### DISCUSSION THREAD

**Discussion by 3point14159** [horizontalsystems/unstoppable-wallet-android#5633] [Ideas]
Sun Knudsen is working on some pretty groundbreaking technology (BlockCrypt) that I think would be of interest to a few of us here.

https://github.com/sunknudsen/blockcrypt

Introducing Blockcrypt, an encryption scheme with plausible deniability by design
https://www.youtube.com/watch?v=bary2XghKjw

Conceptual genesis of BlockCrypt
https://www.youtube.com/watch?v=3SP5CECO5Ik?t=515

The world’s most advanced paper backup hardware device is at your fingertips
https://www.youtube.com/watch?v=2Em7jpxRrrk

His commercial application of the open-source project: https://superbacked.com/

Thoughts?

**Comment by 3point14159** [comment]
I believe this, in combination with Unstoppable Wallet, could make a potent combination for practical security in a world of blockchain'd assets. I would love to coordinate a tutorial/video demonstrating use-cases and a how-to usage. Should be short and simple, and if it isn't, then it's probably not worth doing in the first place, ya know?

**Comment by abdrasulov** [comment]
Thanks for sharing. That is interesting. There is a task called "Duress PIN" #5526. It has a high potential to be included in upcoming releases. We will consider plausible deniability as one way of implementation.

**Comment by 3point14159** [comment]
Looking forward to testing the duress feature. I couldn't create an account on the app center.
